packages/forms-web-app/src/dynamic-forms/validator/string-validator.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars */
+const { body } = require('express-validator');
+const BaseValidator = require('./base-validator.js');
+
+class StringValidator extends BaseValidator {
+	regex = {
+		regex: '',
```

Comment:
Should this be a * so if we don't pass in a regex validation then we are assuming anything is good and that the regex validation is essentially optional?

Reply:
No.

Reply:
A link to the appropriate unit test would be helpful in this case please.

Reply:
I'm more familiar with a wild card regex like the ones here https://howtodoinjava.com/java/regex/match-any-set-of-characters/#3-match-any-character-zero-or-more-occurrences. I think it would be clearer that we're matching anything, including special characters, than an empty string as the default?

Reply:
Please read the unit tests

Reply:
Looks like we're using `''` as falsey (and therefore not including the regex validation in the chain), I generally tend to avoid this (with a preference to explicit false or null) but that's a style preference, it'll work OK.